tests/manual/util/README.md

```diff
@@ -0,0 +1,33 @@
+# Test Utilities
+
+## ManualTestWindow
+
+This facility provides a framework for manual tests to display test instructions, test pane, and Pass/Fail buttons.
+
+A simple test would look like this:
+
+```java
+public class SampleManualTest {
+     public static void main(String[] args) throws Exception {
+         ManualTestWindow.builder().
+             title("Sample Manual Test").
+             instructions(
+                 """
+                 Provide
+                 multi-line instructions here.
+                 """
+             ).
```

Comment:
Add `size(1200, 800).` to example. I think adjusting size will be common enough and no need to look at `ManualTestWindow` to figure it out.

Reply:
good point!